### PR TITLE
LPTOCPCI-1199 - Report success if no tickets created updated or are i…

### PR DESCRIPTION
**Pull request by smatula**
…gnored

### CODE REVIEW

File: src/report/report.py [hunk 1]
```diff
@@ -41,16 +42,18 @@ def __init__(self, firewatch_config: Configuration, job: Job) -> None:
...
-            bugs_filed = self.file_jira_issues(
+            bugs_filed, bugs_updated = self.file_jira_issues(
                 failures=job.failures,  # type: ignore
                 firewatch_config=firewatch_config,
                 job=job,
             )
             if len(bugs_filed) > 1:
                 self.relate_issues(issues=bugs_filed, jira=firewatch_config.jira)
-        else:
+        if (bugs_filed is None or len(bugs_filed) == 0) and (bugs_updated is None or len(bugs_updated) == 0):
```
Comment (amp-rh) [inline src/report/report.py:56]:
```suggestion
        if not bugs_filed and not bugs_updated:
```

Comment (amp-rh) [reply src/report/report.py:56]: Change not required, just a suggestion for readability. Tested change locally with all passing.

**Review by myakove — APPROVED** [review]

**Pull request merged**